lib/Sabre/Location/LocationPhoto.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * @copyright Copyright (c) 2022 Louis Chemineau <louis@chmn.me>
+ *
+ * @author Louis Chemineau <louis@chmn.me>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Photos\Sabre\Location;
+
+use OC\Metadata\FileMetadata;
+use OCA\Photos\DB\Location\LocationFile;
+use OCA\Photos\DB\Location\LocationInfo;
+use OCP\Files\IRootFolder;
+use OCP\Files\Node;
+use OCP\Files\File;
+use OCP\Files\NotFoundException;
+use Sabre\DAV\Exception\Forbidden;
+use Sabre\DAV\IFile;
+
+class LocationPhoto implements IFile {
```

Comment:
PhotoLocation? I suppose it’s the location of a photo and not a photo of a location?

Reply:
Actually, `LocationPhoto` makes sense. It is a photo from the `dav/photos/location` endpoint. Same as `AlbumPhoto`.
Would you name them differently ? Maybe with a `Dav` prefix: `DavLocationPhoto`